cmb2-conditionals.js

```diff
@@ -169,7 +169,7 @@ jQuery( document ).ready( function( $ ) {
 
 		// Selectively trigger the change event.
 		uniqueFormElms = [];
-		$( ':input', changeContext ).each( function( i, e ) {
+		$( '[data-hash]:input', changeContext ).each( function( i, e ) {
```

Comment:
Not sure I understand this change. Could you please ellaborate a bit?

Reply:
The original target will trigger a change event on EVERY input on the page. On our project it triggered non-CMB2  form elements that had JS change events on them. It caused havoc throughout the page. Using this selector, it will target only CMB2 inputs since, it appears, `data-hash` is an attribute on them all.